EventsExpress.Core/Services/PhotoService.cs

```diff
@@ -51,6 +52,42 @@ public async Task<Photo> AddPhoto(IFormFile uploadedFile)
             return photo;
         }
 
+        public async Task<Photo> AddPhotoByURL(string url)
+        {
+            if (!await IsImageUrl(url))
+            {
+                throw new ArgumentException();
+            }
+
+            var client = new HttpClient();
```

Comment:
Please, inject `HttpClient` as a dependency through a constructor. Consider this article: https://docs.microsoft.com/en-us/aspnet/core/fundamentals/http-requests?view=aspnetcore-3.1